verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -22,10 +25,16 @@ export default class CasesController {
 
     /** List simply forwards the request to the data service */
     list = async (req: Request, res: Response): Promise<void> => {
+        if (req.url !== defaultQuery) {
+            logger.info(`Filter applied: ${req.url}`);
+        }
         try {
             const response = await axios.get(
                 this.dataServerURL + '/api' + req.url,
             );
+            if (response.status >= 500) {
+                logger.error(`A server error occurred when trying to list data using URL: ${req.url}. Response status code: ${response.status}`);
+            }
```

Comment:
maybe log anything that isn't a 2xx?

Reply:
I defaulted to 5XX because I wasn't sure whether we wanted to log user errors, but I think 4XX and above makes sense as the web UI really is the user in this case.
I also think ignoring 3XX makes sense, unless we care about redirects or cached responses?

Reply:
ah yes good catch, we probably don't need to log 3xx

Reply:
but 400 is good as we've seen those when we have mismatched deployments